mkdocs_windmill/topbar.html

```diff
@@ -8,7 +8,7 @@
           ><i class="fa fa-search" aria-hidden="true"></i></button>
 
         <div class="input-group input-group-sm wm-top-search">
-          <input type="text" name="q" class="form-control" id="mkdocs-search-query" placeholder="Search">
+          <input type="text" name="q" class="form-control" id="mkdocs-search-query" placeholder="Search" autocomplete="off">
```

Comment:
@ysagal don't understand why this was necessary, AFAIU that's simply your personal preference, no?

Reply:
Because the browser tries to autocomplete the query itself, which looks bad. There may be a more elegant solution, but leaving only windmill to autocomplete seemed like the easiest for now. 
![image](https://user-images.githubusercontent.com/1400013/41908561-5adaa230-7912-11e8-825f-3e868e1645ca.png)

Reply:
Oh, got it, thanks. Haven't had that open for quite a while and, therefore, forgot about the theme autocomplete.